src/app/copmonents/header/header.component.ts

```diff
@@ -0,0 +1,23 @@
+import { resolveSanitizationFn } from '@angular/compiler/src/render3/view/template';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'main-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent {
+
+  private roles = { // till not back-end
+    nobody: '',
+    admin: 'admin',
+    manager: 'manager',
+    recruiter: 'recruiter',
+    interviewer: 'interviewer',
+    mentor: 'mentor'
+  }
```

Comment:
looks like enum for me. Read how it can be used here https://dev.to/shane/working-with-enums-in-angular-html-templates-2io9